docs/src/components/ColorBox.tsx

```diff
@@ -48,7 +48,7 @@ export const colorBoxStyles: ComponentSlotStylesInput<ColorBoxProps, ColorBoxVar
     backgroundColor: p.value,
     border: '1px solid transparent',
     borderRadius: p.rounded && '.25rem',
-    color: Color(p.value).isDark() ? v.colorWhite : v.colorBlack,
+    color: p.value !== undefined && Color(p.value).isDark() ? v.colorWhite : v.colorBlack,
```

Comment:
![image](https://user-images.githubusercontent.com/14183168/56353665-90b42280-61da-11e9-97dd-81cd33a1c541.png)


May it will be better just to skip undefined values?

Reply:
For this I wasn't sure myself. I wanted to emphasize that we are missing some of the colors, but I agree that it is confusing for the community to see these values empty values. @kuzhelov @layershifter what do you think?